roles/libvirt_manager/tasks/manage_vms.yml

```diff
@@ -65,28 +65,6 @@
   register: _ssh_access
   until: _ssh_access.rc == 0
 
-- name: "Configure VM {{ vm }}"
-  when:
-    - vm_type is not match('^(crc|ocp).*$')
-  delegate_to: "{{ vm }}.{{ inventory_hostname }}"
-  remote_user: "{{ _init_admin_user }}"
-  ansible.builtin.shell:
-    executable: /bin/bash
-    cmd: |-
-      test -d /home/zuul && exit 0;
-      set -xe -o pipefail;
-      echo "{{ vm }}" | sudo tee /etc/hostname;
-      sudo hostname -F /etc/hostname;
-      sudo useradd -m -d /home/zuul zuul;
-      echo "zuul ALL=(ALL)  NOPASSWD: ALL" | sudo tee /etc/sudoers.d/zuul;
-      sudo -u zuul mkdir -p /home/zuul/.ssh /home/zuul/src/github.com/openstack-k8s-operators;
```

Comment:
wondering if we shouldn't ensure the `src/...` tree exists via the cloud-init? Or we'll need to make it from some other place, such as the reproducer role. IIRC the `git` module creates the parent tree, but in case we're using something else such as `synchronize` or even plain `rsync` command, we might face an issue if the parent tree doesn't exist.

Reply:
oh frack, i lost the `runcmd` in one of the commits...

Reply:
Note for other reviewers: this directory tree is created in the reproducer role, via the push_code.yml - so we're all good.